projects/plugins/crm/changelog/add-typecheck-script-to-ts-workspaces

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+Added typecheck script to ensure that the TS files are type-checked.
```

Comment:
Probably should either remove the script or update the entry to mention it added a typecheck placeholder.